sdk/src/azure/iot/CMakeLists.txt

```diff
@@ -19,10 +19,18 @@ target_include_directories (az_iot_common
     ${az_SOURCE_DIR}/sdk/inc
 )
 
-target_link_libraries(az_iot_common
+if (APPLE) # temporary
```

Comment:
I would remove the "temporary" comment and ensure this applies to all except Linux for now. (No Windows support for now.)